src/seismometer/__init__.py

```diff
@@ -1,11 +1,22 @@
 import logging
 from pathlib import Path
+from typing import Optional
+
+import pandas as pd
 
 from seismometer._version import __version__
 from seismometer.core.logger import add_log_formatter, set_default_logger_config
 
 
-def run_startup(*, config_path: str | Path = None, output_path: str | Path = None, log_level: int = logging.WARN):
+def run_startup(
+    *,
+    config_path: str | Path = None,
+    output_path: str | Path = None,
+    predictions_frame: Optional[pd.DataFrame] = None,
+    events_frame: Optional[pd.DataFrame] = None,
```

Comment:
```python
pred_df = pd.read_parquet("data/predictions.parquet")
event_df = pd.read_parquet("data/events.parquet")


import seismometer as sm
sm.run_startup(config_path='.', predictions_frame=pred_df, events_frame=event_df, log_level=1)
```

Fails because the dataframe for events uses "EventTime" over "Time"

Is that expected, or should we still listen to the mapping done in Config?

Reply:
Doing the mapping myself does fix the issues

```python
import pandas as pd
pred_df = pd.read_parquet("data/predictions.parquet")
event_df = pd.read_parquet("data/events.parquet")
event_df.rename({"EventTime": "Time"}, inplace=True, axis=1)


import seismometer as sm
sm.run_startup(config_path='.', predictions_frame=pred_df, events_frame=event_df, log_level=1)
```

This does work

Reply:
I think this is the most consistent expectation, that passed frames should be ready for usage.  
There is room to improve the warning messages to make this more apparent, as the transforms after load are likely not known.

Reply:
Can you write up an issue to make the error messages better, more descriptive, suggestive of potential fixes?

Then I'll mark this as resolved

Reply:
Wrote up the issue in #39